src/DTO/Mapper/QuapQuestionnaireMapper.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\DTO\Mapper;
+
+use App\DTO\Model\AspectDTO;
+use App\Entity\Questionnaire;
+use DateTime;
+
+class QuapQuestionnaireMapper { // this name might not be all too fitting
+
+    public static function createFromEntities(array $questionnaires, string $locale, DateTime $dateTime): array {
+        $aspectDTOs = [];
```

Comment:
You mix up the aspects here. If you have to map multiple questionnaires you will end up with a list of aspects without a questionnaire they belong to